src/main/java/randoop/util/ClassFileConstants.java

```diff
@@ -164,16 +180,33 @@ public static ConstantSet getConstants(String classname, ConstantSet result) {
           || c instanceof ConstantUtf8) {
         continue;
       }
+
+      // Here we are examining the constant pool and we increment in the constantToFrequency map
+      // because these are static constants defined in the class.
+      // For example, if our class includes static final int myInt = 31;
+      // but the literal 31 doesn't appear anywhere else in the class, we need to note that
+      // this literal appeared once. This is the place to do so.
+
       if (c instanceof ConstantString) {
-        result.strings.add((String) ((ConstantString) c).getConstantValue(constant_pool));
+        String val = (String) ((ConstantString) c).getConstantValue(constant_pool);
```

Comment:
This is meant to address a line like:
`public static final int mInt = 31;`
that appears in the class but is never used. Although the value would exist in the constant pool part of the class file, there would be no instructions that referenced the value since the variable was unused. I've removed the changes here thinking that such a value wouldn't need to be "constant mined" anyways.